OvmfPkg/Include/Dsc/OvmfTlsLibs.dsc.inc

```diff
@@ -5,7 +5,11 @@
 [LibraryClasses]
   IntrinsicLib|CryptoPkg/Library/IntrinsicLib/IntrinsicLib.inf
 !if $(NETWORK_TLS_ENABLE) == TRUE
+!if $(FD_SIZE_IN_KB) < 4096
   OpensslLib|CryptoPkg/Library/OpensslLib/OpensslLib.inf
+!else
+  OpensslLib|CryptoPkg/Library/OpensslLib/OpensslLibFull.inf

```

Comment:
This is the full but not the accel build. I guess that was the intention, and the PR title is simply inaccurate?

Reply:
First revision switched to full accel, but that failed in CI.  So I dropped the 'accel' part, but didn't update the PR title.  Fixed now.